pages/tasksPage.js

```diff
@@ -0,0 +1,19 @@
+import TaskCard from "@/components/TaskCard";
+import Spinner from "@/components/ui/Spinner";
+import useSWR from "swr";
+
+export default function TasksPage() {
+  const { data: tasksData, isLoading } = useSWR("/api/tasks_items");
```

Comment:
The best practice is to keep API names as short and meaningful as possible. Therefore I suggest:

```suggestion
  const { data: tasksData, isLoading } = useSWR("/api/tasks");
```